dataprofiler/data_readers/data_utils.py

```diff
@@ -272,6 +273,19 @@ def read_json(
     return lines
 
 
+def get_random_number_generator() -> np.random._generator.Generator:
+    """Create a random number generator using a manual seed DATAPROFILER_SEED."""
+    rng = np.random.default_rng(settings._seed)
+    if "DATAPROFILER_SEED" in os.environ and settings._seed is None:
+        seed = os.environ.get("DATAPROFILER_SEED")
+        seed = str(seed)
```

Comment:
I don't believe there's a need to convert to `str`. If it is bc of `os.environ.get("DATAPROFILER_SEED")` returning `None` we can instead set the default value to `""`:

```python
os.environ.get("DATAPROFILER_SEED", "")
```

Reply:
We do need to convert to str, because seed has type Optional[str] rather than str, and therefore cannot be directly loaded into int()

Reply:
Even if you do:
```python
seed: str = os.environ.get("DATAPROFILER_SEED", "")
```
will it not work?

Reply:
fixed following your suggestion!